frontend/src/components/FacilityBooking/ViewBlock.tsx

```diff
@@ -19,15 +21,22 @@ const ViewBlock = (props: Props) => {
   }, [ref.current])
 
   return (
-    <StyledViewBooking ref={ref} isOccupied={props.entry.type === TimeBlockType.OCCUPIED} blockId={props.entry.id}>
-      {props.entry.type === TimeBlockType.OCCUPIED && (
-        <>
+    <>
+      {props.entry.booking ? (
+        <StyledViewBooking
+          ref={ref}
+          isOccupied
+          blockId={props.entry.id}
+          onClick={() => (props.openViewBookingModal(), props.setViewBookingEntryId())}
```

Comment:
is this equivalent to `onClick={() => {
            props.openViewBookingModal();
            props.setViewBookingEntryId()
          }}`?